e2e-tests/portal/cypress/integration/journeys/maker/dashboard/deals/dashboard-table-data.spec.js

```diff
@@ -0,0 +1,246 @@
+const { dashboard } = require('../../../../pages');
+const relative = require('../../../../relativeURL');
+const mockUsers = require('../../../../../fixtures/mockUsers');
```

Comment:
@IgnatG is it worth starting to use module_alias package to make it easier to read the long require ../s?

Reply:
we can definitely look into that. I wouldn't include it in this PR yet, mainly because it would be a separate thing - i.e. we need to start adding stuff to package.json first.
I'll see if I can add it in as part of a separate PR